File: PakClassified.WebApp/PakClassified.WebApp.DTOs/PakClassified.DTOs/AdvertisementImageModel.cs
using a._PakClassified.WebApp.DTOs;
using Microsoft.AspNetCore.Http;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PakClassified.WebApp.DTOs.PakClassified.DTOs
{
    public class AdvertisementImageModel : IEntityNamedModel
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public byte[]? Content { get; set; }
        public IFormFile ContentFile { get; set; }
        public string? Caption { get; set; }
        public string CreatedBy { get; set; }
        public string? LastModifiedBy { get; set; }
        public int AdvertisementId { get; set; }
    }
}

File: PakClassified.Client/src/app/core/models/pakClassified/advertisement-image-model.ts
export interface AdvertisementImagePost {
    id: number;
    name: string;
    contentFile: File;
    caption?: string;
    createdBy: string;
    lastModifiedBy?: string;
    advertisementId: number;
    advertisement?: string;
}

export interface AdvertisementImageGet {
    id: number;
    name: string;
    content: string;
    caption?: string;
    createdBy: string;
    lastModifiedBy?: string;
    advertisementId: number;
    advertisement?: string;
}